earth_enterprise/src/common/CacheSizeCalculations.h

```diff
@@ -19,29 +19,44 @@
 #include "khTypes.h"
 #include <memory>
 #include <vector>
+#include <set>
+#include "notify.h"
+#include <qstring.h>
 
 // determine amount of memory used by generic objects
 template<class T>
 inline uint64 GetObjectSize(const T &obj) {
     return sizeof(obj);
```

Comment:
I think you may want to return 0 here since you will already have a `sizeof(*this)` call in whatever calls this function.